Dockerfile.grafana

```diff
@@ -0,0 +1,54 @@
+FROM grafana/grafana-enterprise:latest
```

Comment:
Rather than depending on latest tag, can we depend on a specific release version?